merge_command.go

```diff
@@ -1,17 +1,34 @@
 package main
 
-import "strings"
+import (
+	"fmt"
+	"net/http"
+	"strings"
+)
 
 const mergingLabel = "merging"
 
 func isMergeCommand(comment string) bool {
 	return strings.TrimSpace(comment) == "!merge"
 }
 
-func handleMergeCommand(issueComment IssueComment, issues Issues) Response {
-	err := addLabel(issueComment.Repository, issueComment.IssueNumber, mergingLabel, issues)
+func handleMergeCommand(issueComment IssueComment, issues Issues, pullRequests PullRequests) Response {
+	errResp := addLabel(issueComment.Repository, issueComment.IssueNumber, mergingLabel, issues)
+	if errResp != nil {
+		return errResp
+	}
+	pr, errResp := getPR(issueComment, pullRequests)
+	if errResp != nil {
+		return errResp
+	} else if *pr.Merged {
+		return SuccessResponse{}
+	} else if !*pr.Mergeable {
+		return SuccessResponse{}
```

Comment:
Function returns success when the PR is not mergeable - is this implemented as intended?